R/Previewer.R

```diff
@@ -122,7 +122,28 @@ reporter_previewer_srv <- function(id,
       })
 
       shiny::observeEvent(input$card_remove_id, {
+        shiny::showModal(
+          shiny::modalDialog(
+            title = "Remove the Report Card",
+            shiny::tags$p(
+              shiny::HTML(
+                sprintf(
+                  "Do you really want to remove <strong>the card %s</strong> from the Report?",
+                  input$card_remove_id
+                )
+              )
+            ),
+            footer = shiny::tagList(
+              shiny::modalButton("Cancel"),
+              shiny::actionButton(ns("remove_card_ok"), "OK", class = "btn-warning")
```

Comment:
I think we should use the same type of buttons as in the [report reset modal](https://github.com/insightsengineering/teal.reporter/blob/1649d79db3f515efca0fa9db79b74fc867022962/R/ResetModule.R#L64-L72).

Otherwise the two modals will be more similar than before but will still be inconsistent.

Reply:
We are waiting for the response and preference from @lcd2yyz now, if we should update the colors in this PR. This PR was blocked because of the suggestion to not use the old colors you suggested to apply.